### PR TITLE
Create database schema - Closes #578

### LINKED ISSUE #578: Create database schema
### Description

Create the database schema. Make sure that data can be written and read from MySQL database through the data layer.

### Motivation

Data persistence is required to achieve reliability of the application.

### Acceptance Criteria

- Database schema reflects our data model
- Tests are written

### Additional information

Property name | Type | Null? | Default | Comment
-- | -- | -- | -- | --
author | varchar(255) | Yes | “LiskHQ” | Blog post author
content_t | text | No | - | Blog post text (truncated)
content_orig | text | No | - | Blog post text (full, unprocessed)
image_url | varchar(255) | Yes | (empty string) | Main picture URL
source | varchar(255) | No | - | Name of the source in the format &lt;provider&gt;_&lt;blog_name&gt;
source_id | varchar(255) | Yes | (empty string) | External ID of the blog post (ie. Twitter ID)
ctime | integer | No | Date.now() | Creation time
mtime | integer | Yes | ctime | Modification time
title | varchar(255) | Yes | (empty string) | Blog post title
url | varchar(255) | No | - | Original URL with the blog post

</div></b>

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #578 

### How was it solved?
- [x] Database schema added
- [x] Support for `utf8mb4` for mysql client
- [x] Tests are written
- [x] Run `newsfeed` service in `Jenkins`
- [x] Run functional tests in `Jenkins`

### How was it tested?
- [x] Local
- [x] Jenkins

### CODE REVIEW

File: services/newsfeed/package.json [hunk 1]
```diff
@@ -27,16 +27,29 @@
...
+    "await-to-js": "^3.0.0",
+    "debug": "^4.3.1",
+    "log4js": "^6.3.0",
+    "moleculer": "=0.13.12",
     "moment": "^2.29.1",
     "object-hash": "^2.2.0",
-    "stack-trace": "^1.0.0-pre1",
+    "pg-promise": "^10.10.2",
+    "require-all": "^3.0.0",
+    "stack-trace": "0.0.10",
```
Comment (MichalTuleja) [inline services/newsfeed/package.json:49]: Please review those dependencies, most of them are not needed. The development-0.5.0 branch has updated package and package-lock files.

File: services/newsfeed/tests/constants/newsfeed.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const newsfeeds = [
```
Comment (MichalTuleja) [inline services/newsfeed/tests/constants/newsfeed.js:16]: `newsfeed` / `news` ?

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

File: services/newsfeed/package.json [hunk 3]
```diff
@@ -27,16 +27,23 @@
...
   },
   "dependencies": {
-    "html-entities": "^2.3.2",
-    "html-to-text": "^1.2.1",
+    "bluebird": "^3.7.2",
+    "knex": "^0.95.4",
+    "mysql": "^2.18.1",
     "lisk-service-framework": "https://github.com/LiskHQ/lisk-service/raw/0ba4de1b8dc37fd35f38ce122321345284f35771/framework/dist/lisk-service-framework-1.2.1.tgz",
     "moment": "^2.29.1",
+    "moleculer": "^0.14.13",
```
Comment (MichalTuleja) [inline services/newsfeed/package.json:39]: This dependency is already imported by the framework, please remove it.

File: services/newsfeed/package.json [hunk 4]
```diff
@@ -27,16 +27,23 @@
...
     "object-hash": "^2.2.0",
-    "stack-trace": "^1.0.0-pre1",
-    "twitter": "^1.7.1"
+    "stack-trace": "0.0.10",
```
Comment (MichalTuleja) [inline services/newsfeed/package.json:41]: Please remove it (if not used) or update it to the recent version.

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

**Review by sameersubudhi — APPROVED** [review]

File: services/newsfeed/package.json (hunk 4 above)
Comment (nagdahimanshu) [reply services/newsfeed/package.json]: Updated!

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**